src/main/java/ChargingRange/ChargingRange.java

```diff
@@ -0,0 +1,31 @@
+package ChargingRange;
+
+import java.util.*;
+
+public class ChargingRange {
+
+    public Map<String, Integer> getRanges(ArrayList<Integer> input) {
+        Collections.sort(input);
+        Map<String, Integer> result = new HashMap<>();
+        int previousElement = input.get(0), currentReference = 0;
+        for (int j = 0; j < input.size(); j++) {
```

Comment:
With the assumption that this is the final code (ignore this comment if this part of code is yet to be implemented), since the first element is already considered, the for loop should be starting from j=1, isnt it? please correct me if I am overlooking anything.

Reply:
Yes, first element is considered but I considered j=0 when array has only one element.